tests/test_biasadjust.py

```diff
@@ -47,11 +47,10 @@ def test_basic_train(self, var, period):
 
     def test_preprocess(self):
 
-        dref360 = self.dref.convert_calendar("360_day", align_on="year")
-
+        dhist360 = self.dhist.convert_calendar("360_day", align_on="year")
```

Comment:
This test failed because ref had s 720 and sim a 730 shape, but actually it should have always failed. The function assumes that the simulation will always always a smaller or equal calendar to the ref ( ref don't have 360 day cal). 
In the previous version, we were "converting" the 360 day cal of the ref to noleap, but the function doesn't add values if you don't specify it ( which we don't because we assume that we are always only removing dates). Hence, it stays 720 long and fails.

In this version, sim is 360 and ref is convert to 360 as well. Both are 720 long

Reply:
The problem was in xclim :  https://github.com/Ouranosinc/xclim/pull/2038/files

I removed the test temporarily.